apps/src/music/MusicView.jsx

```diff
@@ -1,6 +1,8 @@
 /** @file Top-level view for Music */
 import React from 'react';
+import PropTypes from 'prop-types';
 import _ from 'lodash';
+import {Provider, connect} from 'react-redux';
```

Comment:
It would be interesting to think about this connection point in a future architecture!  cc @madelynkasula @molly-moen